frontend/tests/features/wheresMyGene.test.ts

```diff
@@ -178,12 +178,16 @@ describe("Where's My Gene", () => {
 
     await tryUntil(
       async () => {
-        const sourceDataList = await page.$("[class*=MuiList-root]");
```

Comment:
This was selecting a different element than the Source Data List, so I added `data-test-id` on the list to be predictable